packages/terra-clinical-site/src/examples/detail-view/Index.jsx

```diff
@@ -20,9 +20,9 @@ const DetailViewExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2>Detail View</h2>
```

Comment:
lets remove `<h2>Detail View</h2>` as the title is now included in the appropriate props-table